bezier-rs/lib/src/structs.rs

```diff
@@ -0,0 +1,95 @@
+use glam::DVec2;
+use std::fmt::{Debug, Formatter, Result};
+
+/// Struct to represent optional parameters that can be passed to the `project` function.
+#[derive(Copy, Clone)]
+pub struct ProjectionOptions {
+	/// Size of the lookup table for the initial passthrough. The default value is `20`.
+	pub lut_size: i32,
+	/// Difference used between floating point numbers to be considered as equal. The default value is `0.0001`
+	pub convergence_epsilon: f64,
+	/// Controls the number of iterations needed to consider that minimum distance to have converged. The default value is `3`.
+	pub convergence_limit: i32,
+	/// Controls the maximum total number of iterations to be used. The default value is `10`.
+	pub iteration_limit: i32,
+}
+
+impl Default for ProjectionOptions {
+	fn default() -> Self {
+		ProjectionOptions {
+			lut_size: 20,
+			convergence_epsilon: 1e-4,
+			convergence_limit: 3,
+			iteration_limit: 10,
+		}
+	}
+}
+
+/// Struct used to represent the different strategies for generating arc approximations.
+#[derive(Copy, Clone)]
+pub enum MaximizeArcs {
```

Comment:
How about calling this `Strategy`? And then the variants can be `Auto` and perhaps `FavorLargerArcs` and `FavorCorrectness`? The name "maximize arcs" on its own conveys very little understanding to me. When changing it here, remember to change it everywhere else (other functions, local variables within those other functions, comments, wasm wrapper, frontend, etc.)